tests/error/linear_errors/unused_non_terminating.err

```diff
@@ -0,0 +1,10 @@
+Error: Linearity violation (at $FILE:13:8)
+   | 
+11 | 
+12 | @guppy(module)
+13 | def foo(q: qubit @owned) -> None:
+   |         ^^^^^^^^^^^^^^^ Variable `q` with linear type `qubit` is leaked
```

Comment:
Arguably, this shouldn't be an error since non-terminating functions technically cannot leak. However, it would be difficult to generate valid Hugr for cases like this, e.g.

```python
while True:
    qubit()
```

Therefore, I think it's ok to still complain about leaks, even if the function provably doesn't terminate